client/src/planwise/client/projects2/components/dashboard.cljs

```diff
@@ -69,36 +78,38 @@
                           :order @order
-                          :sorted (and (= field @column) (not (nil? @order)))})]
+                          :sorted (and (= field @column) (not (nil? @order)))}
+                         (if (some? tooltip) {:tooltip tooltip}))]
     [ui/sortable-table-header new-props title]))
 
 (defn- scenarios-list
   [scenarios current-project]
   (let [num (count scenarios)
+        source-demand (get-in current-project [:engine-config :source-demand])
         analysis-type (get-in current-project [:config :analysis-type])
         sort-column (rf/subscribe [:scenarios/sort-column])
         sort-order (rf/subscribe [:scenarios/sort-order])
         sorted-scenarios (sort-scenarios scenarios @sort-column @sort-order)]
     [:div.scenarios-content
      [:table.mdc-data-table__table
-      [:caption (generate-title num)]
+      [:caption (generate-title num source-demand)]
       [:thead.rmwc-data-table__head
        [:tr.rmwc-data-table__row.mdc-data-table__header-row
-        [:th ""]
-        [scenarios-sortable-header {:class [:col1] :align :left} "Name" :name]
-        [scenarios-sortable-header {:class [:col2] :align :right} (str (some-> (get-in current-project [:config :demographics :unit-name]) capitalize) " coverage") :demand-coverage]
-        [scenarios-sortable-header {:class [:col5] :align :right} "Geographic Coverage" :geo-coverage]
-        [scenarios-sortable-header {:class [:col6] :align :right} "Population Under Coverage" :population-under-coverage]
-        [scenarios-sortable-header {:class [:col3] :align :right} (if (common/is-budget analysis-type) "Investment" "Effort") :effort]
-        [:th.col4 "Actions"]]]
+        [:th.col0 ""]
+        [scenarios-sortable-header {:class [:col1] :align :left} :name "Name"]
+        [scenarios-sortable-header {:class [:col2] :align :right} :demand-coverage "Population with service" "Population within provider’s catchment area, with access to service provided within capacity."]
+        [scenarios-sortable-header {:class [:col5] :align :right} :without-service "Without service" "Population within provider’s catchment area but without enough capacity to provide service"]
+        [scenarios-sortable-header {:class [:col6] :align :right} :without-coverage "Without coverage" "Population outside catchment area of service providers"]
+        [scenarios-sortable-header {:class [:col3] :align :right} :effort (if (common/is-budget analysis-type) "Investment" "Effort")]
+        [:th.col4 [:p "Actions"]]]]
       [:tbody
-       (map-indexed (fn [index scenario] (scenarios-list-item (:id current-project) scenario index analysis-type)) (concat sorted-scenarios (repeat (- 5 num) nil)))]]]))
+       (map-indexed (fn [index scenario] (scenarios-list-item (:id current-project) scenario index source-demand analysis-type)) (concat sorted-scenarios (repeat (- 5 num) nil)))]]]))
```

Comment:
Probably makes sense to `assoc` the extra props in the scenario to avoid having a very long list of positional arguments. In general, positional arguments should be avoided for clarity. In this case, I'd do something like:
```clojure
(scenarios-list-item (assoc scenario
                            :project-id    (:id current-project)
                            :index         index
                            :source-demand source-demand
                            :analysis-type analysis-type))
```

Reply:
I'll do it.